lms/djangoapps/discussion/views.py

```diff
@@ -312,6 +312,21 @@ def forum_form_discussion(request, course_key):
         return tab_view.get(request, course_id, 'discussion')
 
 
+def redirect_url_to_new_mfe(request, course_id, thread_id):
```

Comment:
What about the base url of the legacy experience? Will that be redirected?
Can you please add unit tests for this? 
And also create a sandbox for testing

Reply:
Base url of legacy experiences will be redirected to MFE.
I will add the test cases.